File: C/Listas/Lista 2/ex08.c
//O custo ao consumidor de um carro novo é a soma do custo de fábrica com a do distribuidor e dos impostos (aplicados ao custo de fábrica).
//Supondo que a percentagem do distribuidor seja de 28% e os impostos de 45%,
//escrever um programa em C que leia o custo de fábrica de um carro e escreva o custo ao consumidor.

#include <stdio.h>
int main(){
    float perc_distribuidor,imposto,custo;
    printf("Informe o custo de fábrica do veículo:");
    scanf("%f", &custo);

    perc_distribuidor=0.28*custo;
    imposto=0.45*custo;

    printf("O valor final do veículo será de R$%.2f",custo+perc_distribuidor+imposto);
    return 0;
}

File: Java/extra/Lista 02/src/ex08.java
/* O custo ao consumidor de um carro novo é a soma do custo de fábrica com a 
percentagem do distribuidor e dos impostos (aplicados ao custo de fábrica). 
Supondo que a percentagem do distribuidor seja de 28% e os impostos de 45%, 
escrever um programa em C que leia o custo de fábrica de um carro e escreva o 
custo ao consumidor. */

import java.util.Scanner;

public class ex08 {
    static double carro(double a, int b, int c) {
        double x = a + (a * ((float) b / 100)) + (a * ((float) c / 100));
        return x;
    }

    public static void main(String[] args) {
        double custo;
        System.out.print("Informe o valor de custo de fábrica do veículo: ");
        try (Scanner leitor = new Scanner(System.in)) {
            custo = leitor.nextDouble();
        }
        System.out.printf("\nO custo ao consumidor será no total R$%.2f", carro(custo, 28, 45));
    }
}